data/data_download/wdpa/Makefile

```diff
@@ -0,0 +1,71 @@
+.PHONY: import
+MAKEFLAGS := --jobs=$(shell nproc)
+MAKEFLAGS += --output-sync=target
+
+URL := $(shell curl -sS 'https://www.protectedplanet.net/downloads' \
+		-H 'content-type: application/json;charset=UTF-8' \
+		--data-raw '{"domain":"general","format":"shp","token":"wdpa","id":51216}' \
```

Comment:
is this `token` prop some kind of unique id? I don't think so as I get the same URL by replacing that with other values, but asking just in case.

Reply:
if so, it would be preferable to pass this in via env vars at runtime